lib/ansible/utils/display.py

```diff
@@ -15,7 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
-from __future__ import (absolute_import, division, print_function)
+from __future__ import (annotations, absolute_import, division, print_function)
```

Comment:
We can just remove the 2.x `__metaclass__` setting and all the other 2.x futures in controller-side code, especially if they're getting updated...